s3-import-commons/src/main/java/no/unit/nva/publication/s3imports/DeleteImportCandidatesEventEmitter.java

```diff
@@ -24,14 +21,15 @@ public class DeleteImportCandidatesEventEmitter implements RequestHandler<S3Even
     public static final String S3_URI_TEMPLATE = "s3://%s/%s";
     public static final String SCOPUS_IDENTIFIER_DELIMITER = "DELETE-";
     public static final int NUMBER_OF_EMITTED_ENTRIES_PER_BATCH = 10;
+    public static final String NOT_EMITTED_EVENTS = "Not emitted events {}:";
     private static final int SINGLE_EXPECTED_FILE = 0;
     private static final Logger logger = LoggerFactory.getLogger(DeleteImportCandidatesEventEmitter.class);
     private final S3Client s3Client;
     private final EventBridgeClient eventBridgeClient;
 
     @JacocoGenerated
     public DeleteImportCandidatesEventEmitter() {
-        this(S3Driver.defaultS3Client().build(), defaultEventBridgeClient());
+        this(S3Driver.defaultS3Client().build(), ApplicationConstants.defaultEventBridgeClient());
```

Comment:
Something is telling me that the "ApplicationConstants" class does a bunch of environment reads. Consider swapping it for your own method if possible.